### DISCUSSION TITLE
Cannot bulk import prefixes via CSV

### DISCUSSION THREAD

**Discussion by awaranowski** [netbox-community/netbox#13327] [Help Wanted!] [answered]
Anyone seen this?  

It won't import prefixes via CSM even if it's in a format that Netbox created.  
For instance, I'll create some prefix entries manually, and then export them as CSV, then delete those prefixes, and attempt to import again.  It will tell me that the first row needs prefix as well as Operational status (both of which are there).  I enabled debugging and logging, and still see nothing.

**Comment by jcollie** [comment]
It's a "design choice" that NetBox won't re-import CSVs that it generated with the export feature. The export uses "human readable" column headers while the import requires "machine readable" column headers that match internal variable names. If you look at any of the import pages it shows you in the table at the bottom what each column header must be.

**Reply by awaranowski** [reply]
Gotcha.  I've looked through the docs, including a page that mentions bulk import using CSV, but I don't see any examples of this.  Do you know of any examples of this?  Haha... I've tried every permutation  I could think of, something like this (these are the only two fields that are required, btw).  I've even tried using JSON.  I keep getting an error saying status and prefix fields are required, when they are provided.

Prefix,Status
76.78.0.0/24,Active

**Reply by jcollie** [reply]
Capitalization matters, try using "prefix" and "status".

**Reply by awaranowski** [reply]
Fantastic, I got it to work.  Everything has to be lowercase.  Little confusing because the inline documentation suggests otherwise.

![image](https://github.com/netbox-community/netbox/assets/4108695/473ce6b2-5f4c-4c21-a352-5d69573920ae)

Thanks!

**Comment by abhi1693** [comment, marked as answer]
Here is a FAQ for this: https://github.com/netbox-community/netbox/wiki/Frequently-Asked-Questions#why-cant-i-pass-exported-data-directly-into-the-import-form

**Reply by awaranowski** [reply]
Thanks.

**Comment by awaranowski** [comment]
I did.
On Tue, Aug 1, 2023 at 5:54 PM Jeffrey C. Ollie ***@***.***>
wrote:

> Capitalization matters, try using "prefix" and "status".
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/netbox-community/netbox/discussions/13327#discussioncomment-6610502>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA7LDF7D3VF7AVQPK5RF4ODXTGCJNANCNFSM6AAAAAA2656DQQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>
-- 
Sincerely,
Andrew
